.github/workflows/Periodic-CI.yml

```diff
@@ -0,0 +1,229 @@
+name: Periodic-CI
+
+on:
+  # On demand execution of workflow will run all suites.
+  workflow_dispatch:
+    inputs:
+      test-targets:
+        required: true
+        description: 'Specify test markers to run'
+        default: '[\"pre_merge\", \"file_system_rules\", \"k8s_object_rules\", \"process_api_server_rules\", \"process_controller_manager_rules\", \"process_etcd_rules\", \"process_kubelet_rules\", \"process_scheduler_rules\"]'
+
+  schedule:
+    # Nightly job. Running every day at 2 am
+    - cron: '00 02 * * *'
+
+    # Weekly job running every saturday at 2 am
+    - cron: '00 02 * * 6'
```

Comment:
will it mean that we're going to trigger the same two jobs on Saturday? also, this is 2 AM UTC which is 5 AM for us, right?

Reply:
So the purpose here is to have something quickly pluggable even if dirty to serve as a reference, to make it easy to add tests to CI as quickly as possible. So it's implemented with the intent that they will be different in the upcoming weeks.

I guess weekly can contain daily checks as a subset, So we can avoid the redundant overlap.